src/commands/test/lib/help-messages.ts

```diff
@@ -0,0 +1,67 @@
+import * as os from "os";
+
+export module Messages {
+  export module TestCloud {
+    export module Commands {
+      export const PrepareAppium = "Creates artifacts directory with Appium tests.";
+      export const PrepareCalabash = `Creates artifacts directory with Calabash tests.${os.EOL}` + 
```

Comment:
Did you know that backtick-strings can be multiline? You could skip the `${os.EOL}` and explicit concatenation and just use a multiline string. No biggie, but you might want to consider for the future.